### PR TITLE
logging improvements for dotnet nuget verify command

### LINKED ISSUE #2359: Update dotnet nuget verify doc as per the spec.

### PR DESCRIPTION
**Pull request by kartheekp-ms**
## Summary
Added a table to explain the details displayed for various verbosity levels of `dotnet nuget verify` command.

Fixes: https://github.com/NuGet/docs.microsoft.com-nuget/issues/2359

### CODE REVIEW

File: docs/core/tools/dotnet-nuget-verify.md
Comment (tdykstra) [inline docs/core/tools/dotnet-nuget-verify.md:53]:
```suggestion
    The following table shows what is displayed for each verbosity level.
```

**Review by tdykstra — COMMENTED** [review]
LGTM, one suggestion for making the intro text more concise.

**Review by tdykstra — APPROVED** [review]
LGTM, do you want to merge now or wait for review by @nuget-team

**Review by heng-liu — APPROVED** [review]

**Pull request merged**
